fastjet/internal/delaunay/delaunay_test.go

```diff
@@ -0,0 +1,293 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package delaunay_test
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+	"time"
+
+	"go-hep.org/x/hep/fastjet/internal/delaunay"
+)
+
+const tol = 1e-3
+
+func TestSimple(t *testing.T) {
+	a := delaunay.NewPoint(0, 0, 0)
```

Comment:
I feel like this would make it very hard to follow when it comes to removing points and checking the results. Do you think I can just leave a comment `// NewPoint(x,y,id)`